runtime/gc_base/GCExtensions.cpp

```diff
@@ -83,11 +89,39 @@ bool
 MM_GCExtensions::initialize(MM_EnvironmentBase *env)
 {
 	PORT_ACCESS_FROM_ENVIRONMENT(env);
+#if defined(LINUX)
+#if defined(J9VM_GC_ENABLE_DOUBLE_MAP)
+	OMRPORT_ACCESS_FROM_J9PORT(privatePortLibrary);
+#endif /* J9VM_GC_ENABLE_DOUBLE_MAP */
+#endif /* LINUX */
 
 	if (!MM_GCExtensionsBase::initialize(env)) {
 		goto failed;
 	}
 
+#if defined(LINUX)
+#if defined(J9VM_GC_ENABLE_DOUBLE_MAP)
+	/* Initialize arraylet hash table */
+	/* Create hash table. Maps arraylet heap addresses to contiguous arraylet leaf addresses */
+	arrayletHashTable = hashTableNew(
+                                   privateOmrPortLibrary,
+                                   J9_GET_CALLSITE(),
+                                   2677,
```

Comment:
Why `2677`?  It might be worth a comment to explain why this particular size was chosen

Reply:
After discussing with Charlie, will change it to be 401 given an average number of objects being kept alive is 250. Unless the application has more than 401 arraylets alive at a given point, this should be enough to avoid collisions and table growth.

Reply:
Cool - good thought basing it on typical applications.  Can you clarify which set of applications used to come to the 401 number?

Reply:
It seems that applications that use more than 100 arraylets are a bit rare. But one in particular, the SPECjbb benchmark, when that uses balanced policy it is not being able to generate the output file due to high use of arraylets (it times out). But I'm not sure how many it would be used in this case.